core/blockly.ts

```diff
@@ -71,8 +71,11 @@ import {Gesture} from './gesture.js';
 import {Grid} from './grid.js';
 import {Icon} from './icon.js';
 import {inject} from './inject.js';
-import {Align, Input} from './input.js';
-import {inputTypes} from './input_types.js';
+import {Align, Input} from './inputs/input.js';
```

Comment:
Is the intention for developers to use `Blockly.inputs.Input` now? Can you fix the comments on the `@see` tags for `ALIGN_LEFT` et al in this file (line 200 or so) if so?

Although, if we do want people to change the import path for `input`, now might be a good time to break the `Align` enum into its own file, so that we can eventually get rid of the `Input` namespace, which is not recommended and makes our documentation more confusing. We could continue to export the new enum from the old location for now.

So new would be
`Blockly.input.Input` and `Blockly.input.Align.LEFT`

and old (deprecated, to be removed) would be `Blockly.Input` and `Blockly.Input.Align.LEFT` aka `Blockly.ALIGN_LEFT`

wdyt? If you don't intend external developers to update their usage then this is all moot, I just think it might be nice to fix while we're here and changing things.

Reply:
I think that's reasonable! That would be a breaking change though correct? If so I think it would be better too do that in a separate PR so we can link to it & include just what broke.

Reply:
I think we should make it a deprecating change, and remove the old stuff in v11. But yes, doing it in its own PR so we can link to it from release notes makes sense.